extensions/podman/package.json

```
@@ -87,7 +87,12 @@
           "format": "file",
           "scope": "ContainerProviderConnectionFactory",
           "description": "Image Path (Optional)"
-        }
+        },
+        "podman.factory.machine.rootful": {
+                  "type": "boolean",
+                  "scope": "ContainerProviderConnectionFactory",
```

```suggestion
                  "scope": "ContainerProviderConnectionFactory",
                  "default": true,
```

If I remember, we wanted to have rootful machines by default (as on macOS/Windows there is already a VM doing isolation) and it works better (less limitation in rootful environment)